src/DiscreteFilter.hpp

```diff
@@ -0,0 +1,134 @@
+#ifndef DISCRETEFILTER_HPP_
+#define DISCRETEFILTER_HPP_
+
+#include <base/samples/Joints.hpp>
+#include <base/samples/RigidBodyState.hpp>
+#include <iostream>
```

Comment:
Ok! Joints and RBS I had to keep in the header because of functions' definitions, but what is not necessary I'll keep in the source code.